luxjs/services/ws.js

```diff
@@ -20,6 +20,18 @@
                 callbacks.push(callback);
             };
 
+            scope.sendMessage = function (url, msg, forceEncode) {
+                if (typeof msg !== 'string' || forceEncode) {
+                    msg = JSON.stringify(msg);
+                }
+                websockets[url].send(msg);
```

Comment:
maybe we need a check if ``websockets[url]`` is available and log an error if not?